### PR TITLE
fix(stack-blitz): default to form-field fill appearance

### PR DESCRIPTION
**Pull request by Splaktar**
Can be tested via https://angularmaterial.dev/components/datepicker/examples

Relates to https://github.com/angular/components/issues/14792 and https://github.com/angular/components/pull/18252.

This should go into `master` ASAP for the v9 final release.

### CODE REVIEW

File: src/assets/stack-blitz/src/main.ts [hunk 1]
```diff
@@ -24,7 +25,9 @@ import {MaterialDocsExample} from './app/material-docs-example';
   entryComponents: [MaterialDocsExample],
   declarations: [MaterialDocsExample],
   bootstrap: [MaterialDocsExample],
-  providers: []
+  providers: [
+    { provide: MAT_FORM_FIELD_DEFAULT_OPTIONS, useValue: { appearance: 'fill' } },
```
Comment (devversion) [inline src/assets/stack-blitz/src/main.ts]: Can you please add a comment on why we do this?

Comment (Splaktar) [reply src/assets/stack-blitz/src/main.ts]: Added, PTAL.

File: README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
 This is the repository for the [Angular Components documentation site](https://material.angular.io/).
 
 Versions of this site are also available for
-- [Beta](https://next.material.angular.io/)
 - [v5](https://v5.material.angular.io/)
 - [v6](https://v6.material.angular.io/)
 - [v7](https://v7.material.angular.io/)
 - [v8](https://material.angular.io/)
+- [v9-rc](https://next.material.angular.io/)
```
Comment (jelbourn) [inline README.md]: Can you revert this? I'm about to update it for actual v9

Comment (Splaktar) [reply README.md]: Done

**Pull request merged**